Userland/Applications/TodoList/Task.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2021, Arthur Brainville <ybalrid@ybalrid.info>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/String.h>
+
+class Task {
+public:
+    enum class State {
+        todo,
+        done
+    };
+
+    static AK::String state_to_string(const State& s)
+    {
+        switch (s) {
+        case State::todo:
+            return "TODO";
+        case State::done:
+            return "DONE";
+        };
+
+        return "";
+    }
+
+    Task(String title, String notes)
+        : title { title }
+        , notes { notes }
+    {
+    }
+    Task(String title)
+        : Task(title, "")
+    {
+    }
+
+    Task() = default;
+
+    State get_state() const;
+    void set_state(State state);
```

Comment:
Getters should not have a `get_` prefix. See https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md

Reply:
removed prefixes